microsoft-edge/webview2/concepts/browser-features.md

```diff
@@ -129,8 +128,8 @@ The following shortcuts are always turned off in WebView2.  An asterisk (`*`) in
 | Select Tab (1 - 8) | `Ctrl`+`(1-8)` |
 | Show Favorites Bar `*` | `Ctrl`+`Shift`+`B` |
 | Help | `F1` |
-| Focus Next Pane `*` | `F6` |
-| Focus Previous Pane `*` | `Shift`+`F6` |
+| Focus Next Pane | `F6` |
```

Comment:
@tofuandeve @jasonstephen15 - in my first commit, I merely removed the * from the F6 row and Shift+F6 row.
Should I remove the two rows, instead?  

See the statement above this table, in the published page
https://docs.microsoft.com/en-us/microsoft-edge/webview2/concepts/browser-features#additional-keyboard-shortcuts-information - 
"The following shortcuts are always turned off in WebView2. An asterisk (*) indicates that the shortcut isn't turned off, but the feature that it accesses is turned off, or the feature doesn't apply to WebView2."

Reply:
From the statement above, if we only remove the * then that would mean F6 is always turned off in WebView2 instead of saying we're partially supporting it now. But completely removing them would also not be very accurate since we are not fully enabling it. I was thinking maybe still leave them here with the * but specify that they are not supported in visual hosting mode? Would that be okay?

Reply:
@tofuandeve - yes that's the same conclusion i came to after reading the page more!  done; kept rows, kept *'s, added an h3 section and linked to it from the two rows.  Added other h3s that the page needed, as well.
@jasonstephen15